app.js

```diff
@@ -1,17 +1,30 @@
 const express = require("express");
 const app = express();
-const { getApiDocumentation } = require("./controllers/app.controller");
+const {
+  getApiDocumentation,
+  getAllTopics,
+} = require("./controllers/app.controller");
 const {
   sqlErrors,
   custumError,
   serverError,
   unmatchRouts,
 } = require("./error.handling");
 
+const updateEndpoits = require("./update_json.util");
+
 app.use(express.json());
 
 app.get("/api", getApiDocumentation);
 
+app.get("/api/topics", getAllTopics);
+
+app.use((req, res, next) => {
```

Comment:
This is an interesting way to link up with your updateEndpoints util file, but you may also start re-writing your endpoints file every time you make a POST or PATCH request as both these conditions would be true (I think - off the top of my head)...